### PR TITLE
PS-8025 Create 8.0.27-18 release notes and release note tasks (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
Merged 5.7 version
M  Makefile
M  source/conf.py
M  source/release-notes/Percona-Server-8.0.27-18.rst
M  source/release-notes/release-notes_index.rst

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+increased visibility. `Commercial support contracts are available
+<https://www.percona.com/services/support/mysql-support>`__.
+
+Release Highlights
+=================================================
+
+The following list are some of the bug fixes for *MySQL* 8.0.27, provided by Oracle, and included in Percona Server for MySQL:
+
+* The ``default_authentication_plugin`` is deprecated. Support for this plugin may be removed in future versions. Use the ``authentication_policy`` variable.
+* The ``binary`` operator is deprecated.  Support for this operator may be removed in future versions. Use ``CAST(... AS BINARY)``.
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-Server-8.0.27-18.rst]: deprecated.  Support -> delete the space

Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Done.

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+* :jirabug:`PS-7960`: Document the ``rocksdb_partial_index_sort_max_mem`` variable, the ``rocksdb_bulk_load_partial_index`` variable, and the ``rocksdb_cancel_manual_compactions`` variable.
+
+
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-7955`: Enable ZenFS functionality on standard Percona Server packages on Debian 11 and Ubuntu 20.04.
+* :jirabug:`PS-7931`: Implement a Slow Query Log rotation in Percona Server for MySQL 8.0.
+* :jirabug:`PS-6730`: In the Slow Query Log, the 'Last_errno:' field should only be used for errors.
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-Server-8.0.27-18.rst]: The 'Last_errno:' field in the Slow Query Log should only contain errors.

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-7883`: An ``ALTER`` query when ``--rocksdb_write_disable_wal`` is enabled causes a server exit.
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-Server-8.0.27-18.rst]: An ``ALTER`` query causes a server exit when ``--rocksdb_write_disable_wal`` is enabled.

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+* :jirabug:`PS-7977`: A broken AppArmor profile after an 8.0.22-13 to 8.0.23-14 upgrade. (Thanks to Alex Kompel for reporting this issue)
+* :jirabug:`PS-2346`: LP #720547: Implement an option to allow queries with specific error codes to add entries to the Slow Query Log.
+* :jirabug:`PS-7958`: A server exit when a ``SELECT`` statement uses a Full-Text Search index with special character.
+* :jirabug:`PS-7940`: Initializing a virtual column template if a child table has a virtual column. This initialization prevents a server exit with an ``ON DELETE CASCADE`` statement in the parent table and the child table had virtual columns. (Upstream :mysqlbug:`105290`)
+* :jirabug:`PS-5654`: On a view, the query digest for each SELECT statement is now based on the SELECT statement and not the view definition, which was the case for earlier versions (Upstream :mysqlbug:`89559`)
+* :jirabug:`PS-7975`: Fix to allow test main.mtr_unit_tests to complete successfully (Thanks to Thomas Deutschmann for reporting this issue)
+* :jirabug:`PS-7873`: Fix for when the log_status table reported an incorrect  executed_gtid (Thanks to zhujzhuo for reporting this issue) (Upstream :mysqlbug:`102175`)
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-Server-8.0.27-18.rst]: an incorrect  executed_gtid - remove an extra space

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+* :jirabug:`PS-5168 Fix for when the Slow Query Log reports ``tmp_table_size:0``.
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Fix for when the Slow Query Log reports - This part is a link now.

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+* :jirabug:`PS-6730`: In the Slow Query Log, the 'Last_errno:' field should only be used for errors.
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-7883`: An ``ALTER`` query when ``--rocksdb_write_disable_wal`` is enabled causes a server exit.
+* :jirabug:`PS-8007`: *Percona Server for MySQL* can also fail to start if the server starts before the network mounts the datadir or local mount of the datadir.
+* :jirabug:`PS-7977`: A broken AppArmor profile after an 8.0.22-13 to 8.0.23-14 upgrade. (Thanks to Alex Kompel for reporting this issue)
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-Server-8.0.27-18.rst]: An AppArmor profile is broken after an 8.0.22-13 to 8.0.23-14 upgrade.

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+* :jirabug:`PS-7931`: Implement a Slow Query Log rotation in Percona Server for MySQL 8.0.
+* :jirabug:`PS-6730`: In the Slow Query Log, the 'Last_errno:' field should only be used for errors.
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-7883`: An ``ALTER`` query when ``--rocksdb_write_disable_wal`` is enabled causes a server exit.
+* :jirabug:`PS-8007`: *Percona Server for MySQL* can also fail to start if the server starts before the network mounts the datadir or local mount of the datadir.
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-Server-8.0.27-18.rst]: *Percona Server for MySQL* can fail

**Comment by breisig** [comment]
What's taking so long to release this version? Especially now that 8.0.28 has been released.

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Fixed

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Done.

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst (hunk 4 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Fixed.

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst (hunk 5 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Unclear.

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst (hunk 6 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Rewritten.

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst (hunk 7 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Removed "also"

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst (hunk 5 above)
Comment (alina-derkach-oaza) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Add `:  after PS-5168

Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Thanks.

File: doc/source/conf.py [hunk 8]
```diff
@@ -45,7 +45,7 @@
 
 # General information about the project.
 project = u'Percona Server'
-copyright = u'Percona LLC and/or its affiliates 2009-2021'
+copyright = u'Percona LLC and/or its affiliates 2009-2022'
```
Comment (LenzGr) [inline doc/source/conf.py]: Good spotting!

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-7883`: An ``ALTER`` query causes a server exit when ``--rocksdb_write_disable_wal`` is enabled.
+* :jirabug:`PS-8007`: *Percona Server for MySQL* can fail to start if the server starts before the network mounts the datadir or local mount of the datadir.
+* :jirabug:`PS-7977`: The AppArmor profile is broken after an 8.0.22-13 to 8.0.23-14 upgrade. (Thanks to Alex Kompel for reporting this issue)
+* :jirabug:`PS-2346`: LP #720547: Implement an option to allow queries with specific error codes to add entries to the Slow Query Log.
```
Comment (LenzGr) [inline doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Thas looks more like a new feature or improvement to me?

File: doc/source/conf.py (hunk 8 above)
Comment (patrickbirch) [reply doc/source/conf.py]: Thanks.

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst (hunk 9 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Confirmed with Oleksander - this is a new feature.

Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Moved to new features.

**Review by alina-derkach-oaza — APPROVED** [review]

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst (hunk 9 above)
Comment (george-lorch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Correct, the JIRA issue type was incorrect.

**Comment by breisig** [comment]
I saw the new percona-server-server-8.0.27-18.1.el8.x86_64 packages in the testing repo but their seems to be conflict with files also in the percona-xtrabackup-80-8.0.27-19.1.el8.x86_64 package [the released version].

```
   file /usr/include/kmip.h from install of percona-xtrabackup-80-8.0.27-19.1.el8.x86_64 conflicts with file from package percona-server-server-8.0.27-18.1.el8.x86_64
   file /usr/include/kmippp.h from install of percona-xtrabackup-80-8.0.27-19.1.el8.x86_64 conflicts with file from package percona-server-server-8.0.27-18.1.el8.x86_64
   file /usr/lib/libkmip.a from install of percona-xtrabackup-80-8.0.27-19.1.el8.x86_64 conflicts with file from package percona-server-server-8.0.27-18.1.el8.x86_64
   file /usr/lib/libkmippp.a from install of percona-xtrabackup-80-8.0.27-19.1.el8.x86_64 conflicts with file from package percona-server-server-8.0.27-18.1.el8.x86_64
```

**Comment by breisig** [comment]
ok I saw the newly generated percona-server-server-8.0.27-18.1.el8.x86_64 packages today in the test repo and now their's no conflict installing percona-xtrabackup-80 :-)

File: doc/source/release-notes/release-notes_index.rst [hunk 10]
```diff
@@ -5,6 +5,7 @@
 .. toctree::
    :maxdepth: 1
 
+   Percona-Server-8.0.27-18
    Percona-Server-8.0.26-16
```
Comment (george-lorch) [inline doc/source/release-notes/release-notes_index.rst:10]: Should 8.0.16-17 be present here?  I do not now for sure, just asking.

**Review by george-lorch — APPROVED** [review]
+1

File: doc/source/release-notes/release-notes_index.rst (hunk 10 above)
Comment (patrickbirch) [reply doc/source/release-notes/release-notes_index.rst:10]: I'll add it.

File: doc/Makefile [hunk 11]
```diff
@@ -54,6 +58,12 @@ html:
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
+
+netlify:
+	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) -c source/conf-netlify $(BUILDDIR)/html
+	@echo
+	@echo "Netlify build finished. The HTML pages are in $(BUILDDIR)/html."
+
```
Comment (LenzGr) [inline doc/Makefile]: I guess that part can be removed and replaced with one for render.com instead, @patrickbirch

**Review by LenzGr — APPROVED** [review]
LGTM. Thanks!

File: doc/Makefile (hunk 11 above)
Comment (patrickbirch) [reply doc/Makefile]: Sure.

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+The following lists a number of the bug fixes for *MySQL* 8.0.27, provided by Oracle, and included in Percona Server for MySQL:
+
+* The ``default_authentication_plugin`` is deprecated. Support for this plugin may be removed in future versions. Use the ``authentication_policy`` variable.
+* The ``binary`` operator is deprecated. Support for this operator may be removed in future versions. Use ``CAST(... AS BINARY)``.
+* Fix for when a parent table initiates a cascading ``SET NULL`` operation on the child table, the virtual column can be set to NULL instead of the value derived from the parent table.
+
+
+Find the full list of bug fixes and changes in the `MySQL 8.0.27 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-27.html>`__.
+
+CentOS 6 is no longer supported.
```
Comment (LenzGr) [inline doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Please change this to:
```suggestion
Red Hat Enterprise Linux 6 (and derivative Linux distributions) are no longer supported. 
```

Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Updated.

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+* :jirabug:`PS-5654`: On a view, the query digest for each SELECT statement is now based on the SELECT statement and not the view definition, which was the case for earlier versions (Upstream :mysqlbug:`89559`)
+* :jirabug:`PS-7975`: Fix to allow test main.mtr_unit_tests to complete successfully (Thanks to Thomas Deutschmann for reporting this issue)
+* :jirabug:`PS-7873`: Fix for when the log_status table reported an incorrect executed_gtid (Thanks to zhujzhuo for reporting this issue) (Upstream :mysqlbug:`102175`)
+* :jirabug:`PS-5168` Fix for when the Slow Query Log reports ``tmp_table_size:0``.
+* Normalized the ``zenfs`` utility backup and restore requirements for the ``--path`` command line option, the ``--backup_path`` command-lne option, and the ``restore_path`` command-line option. For more information, see :ref:`zenfs`.
+
+Known issues
+----------------------------------
+
+The CentOS 7 packages do not support TLS1.3.
```
Comment (LenzGr) [inline doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Please change this to:
```suggestion
RPM packages for Red Hat Enterprise Linux 7 (and compatible derivatives) do not support TLSv1.3, as it requires OpenSSL 1.1.1 (which is currently not available on this platform).
```

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+The following lists a number of the bug fixes for *MySQL* 8.0.27, provided by Oracle, and included in Percona Server for MySQL:
+
+* The ``default_authentication_plugin`` is deprecated. Support for this plugin may be removed in future versions. Use the ``authentication_policy`` variable.
+* The ``binary`` operator is deprecated. Support for this operator may be removed in future versions. Use ``CAST(... AS BINARY)``.
+* Fix for when a parent table initiates a cascading ``SET NULL`` operation on the child table, the virtual column can be set to NULL instead of the value derived from the parent table.
+
+
+Find the full list of bug fixes and changes in the `MySQL 8.0.27 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-27.html>`__.
+
+Red Hat Enterprise Linux 6 (and derivative Linux distributions) are no longer supported.
```
Comment (LenzGr) [inline doc/source/release-notes/Percona-Server-8.0.27-18.rst]: I wonder if that should be a bullet point in the release highlights list above? It looks a bit lost here.

Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Added Packaging Notes below Bug Fixes.

**Comment by LenzGr** [comment]
LGTM! Thank you!

File: doc/source/release-notes/Percona-Server-8.0.27-18.rst (hunk 13 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.27-18.rst]: Done

**Pull request merged**